storybook/preview.tsx

```diff
@@ -28,7 +27,11 @@ export const PageContainer = ({
   <DocsContainer {...props}>
     <Main>
       <Sidebar>
-        <TableOfContents />
+        {/* This is due to Unstyled typed as a being a FC pre-react-18 */}
+        {/* @ts-ignore */}
+        <Unstyled>
```

Comment:
Can this go inside the `TableOfContents.tsx`?

Reply:
Should be able to, will move it and make sure it's all hunky dory

Reply:
Interesting, moving this to the TOC now has a eslist error where it doesn't in preview 🤔 
![Screenshot 2023-03-10 at 2 01 28 pm](https://user-images.githubusercontent.com/36558508/224212819-616fdb78-15d7-457a-939b-70cb83cd3699.png)
I mean we could add it as a dependency, which should clear this up.

Reply:
Just tried and in breaks on the latest alpha, though interestingly if I lock it into `7.0.0-beta.64` like the rest of the beta releases its hunky dory. Maybe they intend to remove Unstyled in alpha .8? Don't know if that will spell trouble for us 🤷